README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/wordpress-mobile/WordPress-iOS-Shared.svg?style=svg)](https://circleci.com/gh/wordpress-mobile/WordPress-iOS-Shared)
+[![Build status](https://badge.buildkite.com/cbe321dd23ff6e5296ff24159934b0c7bc32c49dc5a2fa04c8.svg)](https://buildkite.com/wordpress-mobile/wordpress-shared-ios)
```

Comment:
Same open question about keeping the badge or not as in https://github.com/wordpress-mobile/wpxmlrpc/pull/60#issuecomment-908191601 🤷

Reply:
The badge is gone in https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/298/commits/653d20b36ee1dab638d766f5b9436d6cb60b8acf!